packages/terra-folder-tree/src/subcomponents/FolderTreeItem.jsx

```diff
@@ -61,10 +61,14 @@ const FolderTreeItem = ({
 }) => {
   const theme = useContext(ThemeContext);
 
-  const subfolder = subfolderItems?.length > 0 ? (
+  const [isExpanded, setIsExpanded] = useState(false);
```

Comment:
Shouldn't this be a prop that is set by consumers?  It would match the pattern of our other Terra components.  Also, do we need an isCollapsible prop in the API as well?

Reply:
Looking ahead at Expand All/Collapse All, I think you're right. Will go ahead and make that change.

> Also, do we need an isCollapsible prop in the API as well?

Based on the AC for this Jira, all items with children (AKA folders) should be collapsible, so I don't think we necessarily need a prop for that.